cypress/e2e/Superheroes.cy.ts

```diff
@@ -0,0 +1,30 @@
+describe('Syuperheroes', () => {
+  it('search for superhero', () => {
+    cy.visit('http://localhost:3001');
+
+    cy.contains('A-Bomb');
+    cy.get('input[type=search]').type('Batman');
+    cy.contains('69');
+    cy.contains('Batman');
+  });
+
+  it('change page of superheroes in table for superheroe', () => {
+    cy.visit('http://localhost:3001');
+
+    cy.contains('A-Bomb');
+    cy.get('button').contains('Next').click();
```

Comment:
it would be nice to follow good practices for selecting elements: https://docs.cypress.io/guides/references/best-practices#Selecting-Elements